corehq/apps/userreports/mixins.py

```diff
@@ -75,6 +75,29 @@ def inner_columns(self):
             for inner_col in col.get_column_config(self.config, self.lang).columns
         ]
 
+    @property
+    def inner_columns_for_display(self):
+        """
+        This returns a list of dicts of column_id and corresponding display for all columns
+        that would appear in the data pulled from the data source
+        """
```

Comment:
Built on the same principle as used in https://github.com/dimagi/commcare-hq/blob/c156b8b9cd69f0043624f633c0fd8bcfeaa2a949/corehq/apps/userreports/sql/columns.py#L25 for data_rows at https://github.com/dimagi/commcare-hq/blob/c3468e680250c253c4bf17123a2d16ed0d53008d/corehq/apps/userreports/reports/util.py#L82